ReadMe.md

```diff
@@ -136,6 +136,15 @@ For the Asp.NET projects, including the core library, you can run on the latest
 
 For the Maui sample project, you need the latest [_preview_ of Visual Studio](https://visualstudio.microsoft.com/vs/preview/).
 
+If you have not installed node.js on your system, you will need to download and install it in order for the npm scripts to run. Please select
+the appropriate installer for your system (https://nodejs.org/en/download/).
+
+If you have not installed powershell on your system (macOS and Linux users), you will need to install it and change the "execution policies" on the system.
```

Comment:
I would add, "or if you would like the latest version of Powershell for Windows"...